README.md

```diff
@@ -72,6 +72,80 @@ resource "aws_launch_configuration" "multipart" {
 }
 ```
 
+### Example with using the iam_policy_document and aws_iam_role:
+
+```hcl
+locals {
+  application {
+    name      = "cloudwatch_agent"
+    stage     = "dev"
+    namespace = "eg"
+  }
+}
+
+module "label" {
+  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
+  environment = "${local.application["stage"]}"
```

Comment:
why `environment` and not `stage`?

Reply:
Same as before I am just used to using environment and it is a kinda of a muscle memory :)